src/openrct2/ride/Ride.h

```diff
@@ -1109,6 +1109,8 @@ int32_t RideGetEntryIndex(int32_t rideType, int32_t rideSubType);
 void DetermineRideEntranceAndExitLocations();
 void RideClearLeftoverEntrances(const Ride& ride);
 
+void SetBrakeClosed2(TrackElement& trackElement, const CoordsXY& trackLocation, bool isClosed);
```

Comment:
Could you go for a bit more descriptive name. SetBrakeMultiTileClosed would make it clear that this sets all the track elements in the track block. I know it's also doing single tile but that's just an optimisation.